RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -24,6 +24,7 @@ namespace PFTrackAlgoTools {
       case reco::TrackBase::pixelLessStep:
         return cuts[3];
       case reco::TrackBase::tobTecStep:
+      case reco::TrackBase::siStripTripletStep:
```

Comment:
I'm not super sure this change belongs to the TRK POG responsability
and actually it has not mentioned during the TRK POG meeting

since I'm not super sure this is the way to go,
I let @bendavid and @hatakeyamak to comment on this (thanks !)